src/blocks/CallToAction/index.module.scss

```diff
@@ -1,43 +1,54 @@
 @use '@scss/common.scss' as *;
 
-.callToAction {
+.wrapper {
   position: relative;
 }
 
-.bgWrapper {
-    width: 100%;
-    height: 100%;
-    position: absolute;
-    top: 0;
-    left: 0;
-}
+.container {
+  position: relative;
+  padding-top: calc(var(--base) * 10);
+  padding-bottom: calc(var(--base) * 10);
```

Comment:
@paulpopus `padding` here is too big in mobile - let's decrease it for `mid` / `small` breakpoints

![Screen Shot 2024-01-30 at 10 58 51 AM](https://github.com/payloadcms/website/assets/35232443/806d3486-82ae-434b-9e0c-5c03906881a9)